src/ovp_pipeline/truth_api.py

```
@@ -2871,12 +2871,50 @@ def _focused_action_precondition(
+    action_kind = str(action.get("action_kind") or "")
+    if action_kind != "object_extraction_workflow":
+        return {"status": "ready", "reason": ""}
+    pack_name = str(action.get("pack") or DEFAULT_WORKFLOW_PACK_NAME)
+    for note_path in _focused_action_note_paths(action):
+        try:
+            traceability = get_note_traceability(
+                vault_dir,
+                note_path=note_path,
+                pack_name=pack_name,
+            )
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)